src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -139,7 +139,7 @@ public function resolvePackages(array $packagesToRequire): array
                 continue;
             }
 
-            $packagesToRequire[] = new PackageRequireOptions($packageName.$cssFile, $version, $options->download);
+            $packagesToRequire[] = new PackageRequireOptions($package.$cssFile, $version, $options->download);
```

Comment:
Line 85 should also be `$packageName` I think (for the array key). Because `$options->packageName` could actually be a package + path - like `chart.js/auto` . It doesn't matter in practice, because we only request the CSS entrypoint when installing a pure package (with no path), but we might as well fix that up too.